docs/src/content/docs/reference/utilities/get-locale-path.mdx

```diff
@@ -8,14 +8,25 @@ Allows you get the right url for the current locale. It's strictly typed and eve
 `getLocalePath` does not take your `trailingSlash` configuration into account and always returns without a trailing slash.
 :::
 
-```astro
+```astro title="/fr/my-page"
```

Comment:
This is an astro file right?
```suggestion
```astro title="/fr/my-page.astro"
```

Reply:
Actually I wanted to pretend this is the current url, but it's not clear enough